projects/driver_controls/inc/steering_wheel.h

```diff
@@ -0,0 +1,42 @@
+#pragma once
+
+// Module for steering wheel angle percentage reading
+//
+// The goal of this module is to convert the analog reading of a rotary
+// sensor to an accurate percentage output. This will be accomplished by
+// using the ADC to convert the sensor's analog input to digital input, and
+// using calibrated data from the steering wheel calibration module to accurately
+// convert digital input (on a scale of 0 to 4095) to a scale of (-100% to 100%),
+// where -100% is the counterclockwise-most position of the wheel, and
+// 100% is the clockwise-most position of the wheel.
+//
+// Requires steering wheel calibration module
```

Comment:
This module does not require the calibration module to operate. It should not have to know where the data is coming from.